platform-hub-web/src/app/projects/projects-detail.html

```diff
@@ -115,27 +116,31 @@ <h4 class="md-subhead centred">Team members</h4>
                 <md-menu-content width="4">
                   <md-menu-item ng-if="$ctrl.allowOnboardOrOffboardGitHub(m)">
                     <md-button
+                      ng-if="m.user.is_active"
                       ng-click="$ctrl.userOnboardGitHub(m.user.id, $event)"
                       aria-label="Onboard this user to the GitHub org and team">
                       Onboard GitHub
                     </md-button>
                   </md-menu-item>
                   <md-menu-item ng-if="$ctrl.allowOnboardOrOffboardGitHub(m)">
                     <md-button
+                      ng-if="m.user.is_active"
```

Comment:
If all these actions in the menu are turned off when the user is not active, then maybe we should just turn off the the whole menu in that case? (unless there are specific actions the PM/admin needs to be able to take on deactivated users?)